.github/workflows/build.yml

```diff
@@ -133,6 +133,7 @@ jobs:
           echo "CFLAGS=$CFLAGS_${{ matrix.arch }}" >> $GITHUB_ENV
           echo "LDFLAGS=$LDFLAGS_${{ matrix.arch }}" >> $GITHUB_ENV
       - uses: actions/checkout@v3
+      - uses: ros-tooling/setup-ros@v0.6
```

Comment:
This looks interesting, but I'm not clear on what exactly it's doing(?)

Reply:
Scrub the above. I see you're looking at the build issues. I'm still haven't fixed my email notifications on gitter.